web/src/main/java/it/pagopa/selfcare/pagopa/backoffice/web/controller/InstitutionController.java

```diff
@@ -85,8 +85,8 @@ public void regeneratePrimaryKey(@ApiParam("${swagger.model.institution.id}")
     @PostMapping("/{institutionId}/api-keys/secondary/regenerate")
     @ResponseStatus(HttpStatus.NO_CONTENT)
     @ApiOperation(value = "", notes = "${swagger.api.institution.regenerateSecondaryKey}")
-    public void regenerateSecondaryKey(@ApiParam("${swagger.model.institution.id}")
-                                     @PathVariable("institutionId")String institutionId){
+    public void regenerateSecondaryKey(@ApiParam("${swagger.model.institution.subscription.id}")
+                                       @PathVariable("institutionId") String institutionId) {
```

Comment:
swagger description changed to subscription Id but path variable stayed institutionId